src/modules/menu/components/RecommendedFoodCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { type Food } from "@/modules/mock/foods";
+import styled from "@emotion/styled";
+import { Col, ConfigProvider, Tag, Typography } from "antd";
+import Image from "next/image";
+
+type RecommendedFoodCardProps = {
+  food: Food;
+};
+
+const RecommendedFoodCard: React.FC<RecommendedFoodCardProps> = ({ food }) => {
+  const {
+    id,
+    name = "RecommededFoodImage",
+    price,
+    description,
+    imagePath,
+  } = food;
+
+  return (
+    <Col span={12}>
+      {imagePath && (
+        <StyledImage src={imagePath} alt={name} width={200} height={200} />
+      )}
+      <ConfigProvider
+        theme={{
+          components: {
+            Typography: {
+              colorText: "#ffffff",
+            },
+          },
+        }}
+      >
+        <FoodNameText>{name}</FoodNameText>
+      </ConfigProvider>
+      <PriceTag>
+        <PriceText>{price}฿</PriceText>
```

Comment:
We must talk about what unit we use ( ฿ or THB or BAHT)

Reply:
From KATID use Baht before disgust from design team

Reply:
Done